llm_on_ray/finetune/finetune.py

```diff
@@ -193,92 +242,43 @@ def train_func(config: Dict[str, Any]):
         }
     )
 
-    optimizer = common.optimizer.Optimizer.registory.get("DefaultOptimizer")()(
-        model,
-        config={
-            "name": config["Training"]["optimizer"],
-            "config": {"lr": config["Training"]["learning_rate"]},
-        },
+    data_collator = common.dataprocesser.general_processer.DataCollatorForCompletionOnlyLM(
+        tokenizer=tokenizer, mlm=False, return_tensors="pt", pad_to_multiple_of=8
     )
 
-    accelerate_mode = config["Training"]["accelerate_mode"]
-    if accelerate_mode == "FSDP":
-        fsdp_plugin = FullyShardedDataParallelPlugin(
-            state_dict_config=FullStateDictConfig(offload_to_cpu=False, rank0_only=False),
-            optim_state_dict_config=FullOptimStateDictConfig(
-                offload_to_cpu=False, rank0_only=False
-            ),
```

Comment:
transformers.Trainer how to distinguish FSDP and Deepspeed?

Reply:
for FSDP training, we should change TrainingArguments's option as following:
`fsdp_config` with a json config file: fsdp_config.json
`fsdp auto_wrap` set True